docs/usage-guide/validators.md

```diff
@@ -45,9 +45,9 @@ enter an email address again (hopefully a valid one this time).
 ```
 
 [^1]:
```

Comment:
There are three, total references to this footnote in this [file](https://github.com/wayfair-incubator/columbo/blob/ef72659f086229a7853d381ad64ddf841f67ff5a/docs/usage-guide/validators.md#validator-structure). I recommend removing the first two references to this footnote as they are no longer relevant.

Reply:
I feel like the footnote can be removed entirely. Then rephrase the `Upgrading Validator Structure` section to mentioned that "`Optional[str]` was used <0.10.0 and was removed on 1.0.0"

Reply:
Good point - since the docs are versioned, we can remove this footnote and section entirely.

Reply:
Cool updated to remove